applications/Chat/coati/experience_maker/base.py

```diff
@@ -64,13 +64,26 @@ def __init__(self,
                  critic: nn.Module,
                  reward_model: nn.Module,
                  initial_model: Actor,
-                 kl_coef: float = 0.1) -> None:
+                 kl_coef: float = 0.1,
+                 max_length: int = 128,
```

Comment:
These are all generation kwargs. Is it essential to declare these kwargs here explicitly?